FinancialTracker/ViewControllers/TabBarController.swift

```diff
@@ -114,9 +104,9 @@ class TabBarController: UITabBarController {
                 case let currencyVC as CurrencyTableViewController:
                     currencyVC.authManager = authManager
                     currencyVC.navigationItem.rightBarButtonItem = signOutButton
-                case let premiumVC as PremiumViewController:
-                    premiumVC.authManager = authManager
-                    premiumVC.navigationItem.rightBarButtonItem = signOutButton
+                case let remindersVC as RemindersTableViewController:
+                    remindersVC.authManager = authManager
+                    remindersVC.navigationItem.rightBarButtonItem = signOutButton
                 default:
                     break
```

Comment:
put some kind of `assetionFailure` in the default case to catch cases that are not handled in the future.